src/client/src/App/Credit/common/timers.ts

```diff
@@ -0,0 +1,17 @@
+import { RfqState } from "@/generated/TradingGateway"
+import { RfqDetails } from "@/services/credit"
+
+function getRfqRemainingTime(rfq: RfqDetails): number {
+  return Date.now() - Number(rfq.creationTimestamp) + rfq.expirySecs * 1000
+}
+
+export function timeRemainingComparator(
+  rfq1: RfqDetails,
+  rfq2: RfqDetails,
+): number {
+  return getRfqRemainingTime(rfq1) - getRfqRemainingTime(rfq2)
+}
+
+export function isRfqTerminated(rfqState: RfqState): boolean {
```

Comment:
not really a "timers" thing .. as you have killed the utils.ts, perhaps put in the main `common/index.ts`